tests/model/test_contract.py

```diff
@@ -122,3 +125,28 @@ def test_file_definition_task_no_required_args(data_contract_data: Dict):
     tasks.append({"name": "set_nan_to_none", "params": None})
     d: DataContract = DataContract(**data_contract_data)
     assert d is not None
+
+
+def test_datacontract_with_invalid_external_config(
```

Comment:
are we missing a test for data contract with external config that is successful?

Reply:
The test for a successful external config is here: https://github.com/LinuxForHealth/CsvToFHIR/blob/distributed_datacontract/tests/test_converter.py#L388. Its a more thorough test that checks both the successful loading and successful conversion of the csv all-in-one. 

Having said that I think a more directed test at the functionality is a good idea. so I added one.